### PR TITLE
feat: add vertical tab button for new home page exp

### PR DESCRIPTION
**Pull request by christian14b**
This pr adds a vertical tab button for [Home Page Experiment component](https://github.com/kiva/ui/pull/4162)

image

### CODE REVIEW

**Review by dyersituations — APPROVED** [review]

**Comment by eddieferrer** [comment]
@mcstover @dyersituations 

I think that semantically, this is not a button and this is overloading the button component with too many variations. 

I would much rather see this implemented as a vertical variation of the KvTabs component. That supports a set of 1 tab (looks attached picture) or even multiple vertical tabs.

**Comment by dyersituations** [comment]
> @mcstover @dyersituations
> 
> I think that semantically, this is not a button and this is overloading the button component with too many variations.
> 
> I would much rather see this implemented as a vertical variation of the KvTabs component. That supports a set of 1 tab (looks attached picture) or even multiple vertical tabs.

Good call, and I guess it would depend on the full use case. That's part of why I'm always hesitant to approve PRs without the full context. The code here seems fine, but without seeing the design/AC, it's hard to know if this is a button vs a tab vs something else entirely.

**Comment by mcstover** [comment]
@eddieferrer I think that's a valid comment. @christian14b What do you think the lift would be to make alterations to the tabs component?

**Comment by eddieferrer** [comment]
Looks great, don't forget to clean up KvButton and story

**Review by eddieferrer — APPROVED** [review]

File: @kiva/kv-components/vue/stories/KvTabs.stories.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ import KvTabPanel from '../KvTabPanel.vue';
 export default {
 	title: 'KvTabs',
 	component: KvTabs,
+	args: {
+		vertical: true,
```
Comment (dyersituations) [inline @kiva/kv-components/vue/stories/KvTabs.stories.js:9]: I'm guessing you don't want vertical for all of the stories. If so, add to the `VerticalOrientation` story below.

**Review by dyersituations — APPROVED** [review]

File: @kiva/kv-components/vue/stories/KvTabs.stories.js (hunk 1 above)
Comment (mcstover) [reply @kiva/kv-components/vue/stories/KvTabs.stories.js:9]: Agree, please verify that behavior and appearance of the existing stories.

Comment (mcstover) [reply @kiva/kv-components/vue/stories/KvTabs.stories.js:9]: Well, it looks like our other existing stories aren't "applying" the default setting to their props. Thus, there were no changes to these stories in the Chromatic diff.

**Review by mcstover — APPROVED** [review]

**Pull request merged**